yt/fields/particle_fields.py

```diff
@@ -847,33 +847,6 @@ def _pfunc_avg(field, data):
     return fn
 
 
-def add_volume_weighted_smoothed_field(
```

Comment:
Aren't we supposed to keep this one until `yt 4.0.1`?

Reply:
that's open for discussion. IMO, keeping it around after removing the body of the function doesn't make much sense, so it should either be removed completely or restored to its real form for one more minor release.